students/jemielity_kamil/lesson_06_dicts_tuples_sets_args_kwargs/frequency_analysis.py

```diff
@@ -0,0 +1,24 @@
+
+Dictionary = {}
```

Comment:
Dictionary -> lowercase
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Reply:
fixed